include/utils.h

```diff
@@ -112,5 +114,20 @@ randomCategoriesLimits(int nbCategories, unsigned long int seed = time(NULL)) {
   sort(catLimits.begin(), catLimits.end());
   return catLimits;
 }
+/**
+ * Overloading << operator for std::unordered_map
+ *
+ * @param out ostream
+ * @param std::unordered_map  object
+ *
+ */
+template <typename K, typename V>
+std::ostream &operator<<(std::ostream &out, std::unordered_map<K, V> const &m) {
+  for (auto const &pair : m) {
+    out << "{, (" << pair.first << ": " << pair.second << "),";
```

Comment:
Either we need to remove this if we don't feel like using it, or we need to adapt the << operator in alternativePerformance so that we use it and it matches in the tests

Reply:
Did overwrite the << operator to include this unordered map printing